src/drivers/optical_flow/paw3902/paw3902_main.cpp

```diff
@@ -65,19 +62,18 @@ I2CSPIDriverBase *PAW3902::instantiate(const BusCLIArguments &cli, const BusInst
 	return instance;
 }
 
-int
-paw3902_main(int argc, char *argv[])
+extern "C" __EXPORT int paw3902_main(int argc, char *argv[])
 {
-	int ch;
+	int ch = 0;
 	using ThisDriver = PAW3902;
 	BusCLIArguments cli{false, true};
 	cli.spi_mode = SPIDEV_MODE0;
-	cli.default_spi_frequency = PAW3902_SPI_BUS_SPEED;
+	cli.default_spi_frequency = SPI_SPEED;
 
-	while ((ch = cli.getopt(argc, argv, "R:")) != EOF) {
+	while ((ch = cli.getopt(argc, argv, "Y:")) != EOF) {
```

Comment:
Is there any benefit to not keeping the original `-R` convention used in all other drivers? I'm also thinking of future use if/when we support arbitrary flow sensor orientations.

Reply:
Not that I could think of. 

I'm hoping we can make this all configurable per instance "soon". https://github.com/PX4/PX4-Autopilot/issues/15683

Reply:
Then let's revert this to keep in sync with the existing style.

Reply:
Oh sorry, I misread your comment as "is there any benefit to keeping the original -R".

1. We currently only support flow facing down.
2. People are doing things like putting these on the arm of a multicopter which might not be at a 45 degree multiple.

Reply:
Ah, kk.